src/vivarium/framework/results/manager.py

```diff
@@ -236,7 +236,7 @@ def register_observation(
     def _add_resources(self, target: List[str], target_type: SourceType):
         if not len(target):
             return  # do nothing on empty lists
-        target = set(target) - {"event_time", "current_time", "step_size"}
+        target = set(target) - {"event_time", "current_time", "event_step_size"}
```

Comment:
Is this the "global" step-size? If so, is it used by any existing observers? Is there a compelling reason that an observer would need this?

Reply:
It's the step size that comes from the event (which is defined by the global time ofc). I don't know of any observations/stratifications that use it, but "event_time" is in a commented out piece of stratification code
https://github.com/ihmeuw/vivarium_public_health/blob/0703d0964d6b92d99bb818db25802e8364fcd266/src/vivarium_public_health/metrics/stratification.py#L54
I preserved it just to be on the safe side, but happy to drop it altogether if it is not needed